src/Tester/UnitTestSiloHost.cs

```diff
@@ -25,24 +28,13 @@ namespace UnitTests.Tester
     [DeploymentItem("OrleansProviders.dll")]
     [DeploymentItem("TestInternalGrainInterfaces.dll")]
     [DeploymentItem("TestInternalGrains.dll")]
-    public class UnitTestSiloHost : TestingSiloHost
+    public abstract class UnitTestSiloHost
     {
-        public UnitTestSiloHost() // : base()
-        {
-        }
-
-        public UnitTestSiloHost(bool startFreshOrleans)
-            : base(startFreshOrleans)
-        {
-        }
+        protected static readonly Random random = new Random();
```

Comment:
Probably better to use `SafeRandom` here, just to ensure there are no unforeseen glitches if any test cases try to use random inside concurrent executing closures etc.